torch/_inductor/compile_fx.py

```diff
@@ -324,8 +325,7 @@ def cudagraphify(
     def run(new_inputs):
         nonlocal compiled_fn
         if compiled_fn is None:
-            with dynamo_utils.preserve_rng_state():
```

Comment:
how come?

Reply:
Ughhh actually I can revert this, since I'm doing the deferred cudagraphify. Otherwise it would make it so the rng was reset even for the cuda graph replay run. 

I added extra tests for rng for new and old cudagraphs